packages/appstore/src/components/main-title-bar/account-type-dropdown.scss

```diff
@@ -38,34 +38,34 @@
     &--real,
     &--real:focus,
     &--real:hover {
-        border: 1px solid var(--status-info);
+        border: 1px solid var(--status-success);
         width: 87%;
         & > .dc-text {
-            color: var(--status-info);
+            color: var(--status-success);
         }
     }
 
     &--demo,
     &--demo:focus,
     &--demo:hover {
-        border: 1px solid var(--status-success);
+        border: 1px solid var(--status-info);
         width: 90%;
         & > .dc-text {
-            color: var(--status-success);
+            color: var(--status-info);
             padding-left: 0.8em;
         }
     }
     &__icon {
         &--real,
         &--demo {
-            transform: none !important;
+            transform: none;
```

Comment:
```suggestion
            &__icon {
            transform: none;
```

Reply:
Can we refactor it to this?

`&__icon {
        tranform: none;
        &--real {
            --fill-color1: var(--status-info) !important;
            right: 1.8rem;
        }
        &--demo {
            --fill-color1: var(--status-success) !important;
            right: 1.5rem;
        }
    }`

Reply:
refactored and pushed to the same PR as above @hirad-rewok